File: android/src/main/java/com/lightningkite/kwift/observables/binding/SelectDateRangeView.binding.actual.kt
package com.lightningkite.kwift.observables.binding

import com.lightningkite.kwift.observables.*
import com.lightningkite.kwift.views.android.SelectDateRangeView
import java.util.*

fun SelectDateRangeView.bind(start: MutableObservableProperty<Date?>, endInclusive: MutableObservableProperty<Date?>) {
    this.start = start.transformed(
        read = { it?.let { Calendar.getInstance().apply { timeInMillis = it.time } } },
        write = { it?.time }
    )
    this.endInclusive = endInclusive.transformed(
        read = { it?.let { Calendar.getInstance().apply { timeInMillis = it.time } } },
        write = { it?.time }
    )
}

File: ios/Kwift/com/lightningkite/kwift/observables/binding/SelectDateRangeView.binding.actual.swift
//Stub file made with Kwift 2 (by Lightning Kite)
import Foundation


//--- SelectDateRangeView.bind(MutableObservableProperty<Date?>, MutableObservableProperty<Date?>)
public extension SelectDateRangeView {
    func bind(_ start: MutableObservableProperty<Date?>, _ endInclusive: MutableObservableProperty<Date?>) -> Void {
        self.start = start
        self.endInclusive = endInclusive
    }
    func bind(start: MutableObservableProperty<Date?>, endInclusive: MutableObservableProperty<Date?>) -> Void {
        return bind(start, endInclusive)
    }
}









